diff --git a/include/liquid.h b/include/liquid.h
index b946ee81..2c631752 100644
--- a/include/liquid.h
+++ b/include/liquid.h
@@ -150,11 +150,11 @@ LIQUID_DEFINE_COMPLEX(double, liquid_double_complex);
 
 // external compile-time deprecation warnings with messages
 #ifdef __GNUC__
-#   define DEPRECATED(MSG,X) X __attribute__((deprecated (MSG)))
+#   define LIQUID_DEPRECATED(MSG,X) X __attribute__((deprecated (MSG)))
 #elif defined(_MSC_VER)
-#   define DEPRECATED(MSG,X) __declspec(deprecated) X
+#   define LIQUID_DEPRECATED(MSG,X) __declspec(deprecated) X
 #else
-#   define DEPRECATED(MSG,X) X
+#   define LIQUID_DEPRECATED(MSG,X) X
 #endif
 
 //
@@ -963,7 +963,7 @@ int EQLMS(_copy_coefficients)(EQLMS() _q,                                   \
 /* Get equalizer's internal coefficients                                */  \
 /*  _q      : filter object                                             */  \
 /*  _w      : pointer to output coefficients array [size: _n x 1]       */  \
-DEPRECATED("use eqlms_xxxt_copy_coefficients(...) instead",                 \
+LIQUID_DEPRECATED("use eqlms_xxxt_copy_coefficients(...) instead",                 \
 void EQLMS(_get_weights)(EQLMS() _q,                                        \
                          T *     _w)                                        \
 );                                                                          \
@@ -1032,7 +1032,7 @@ int EQLMS(_step_blind)(EQLMS() _q,                                          \
 /*  _x      :   received sample vector,[size: _n x 1]                   */  \
 /*  _d      :   desired output vector, [size: _n x 1]                   */  \
 /*  _n      :   input, output vector length                             */  \
-DEPRECATED("method provides complexity with little benefit",                \
+LIQUID_DEPRECATED("method provides complexity with little benefit",                \
 int EQLMS(_train)(EQLMS()      _q,                                          \
                   T *          _w,                                          \
                   T *          _x,                                          \
@@ -5460,7 +5460,7 @@ int gmskframegen_write(gmskframegen           _q,
 // write samples of assembled frame
 //  _q          : frame generator object
 //  _buf        : output buffer [size: k x 1]
-DEPRECATED("use gmskframegen_write(...) instead",
+LIQUID_DEPRECATED("use gmskframegen_write(...) instead",
 int gmskframegen_write_samples(gmskframegen           _q,
                                liquid_float_complex * _buf)
 );
@@ -5501,11 +5501,11 @@ int              gmskframesync_reset_framedatastats(gmskframesync _q);
 framedatastats_s gmskframesync_get_framedatastats  (gmskframesync _q);
 
 // debug methods
-DEPRECATED("debug methods add complexity and provide little value",
+LIQUID_DEPRECATED("debug methods add complexity and provide little value",
   int gmskframesync_debug_enable(gmskframesync _q) );
-DEPRECATED("debug methods add complexity and provide little value",
+LIQUID_DEPRECATED("debug methods add complexity and provide little value",
   int gmskframesync_debug_disable(gmskframesync _q) );
-DEPRECATED("debug methods add complexity and provide little value",
+LIQUID_DEPRECATED("debug methods add complexity and provide little value",
   int gmskframesync_debug_print(gmskframesync _q, const char * _filename) );
 
 
